docs/planning-experiments/index.md

```diff
@@ -0,0 +1,6 @@
+# Planning Experiments
+
+- [Minimum Detectable Effects](./minimum_detectable_effects)
+- [Entry Points](./entry_points)
+- [Setting up the Sample Size Calculator](./setting_up_the_sample_size_calculator)
+- [Using the Sample Size Calculator](./using_the_sample_size_calculator)
```

Comment:
I think it'd be great if this page has a bit more content that gives the summary/TLDR of what's in other pages

Reply:
Added this!